P1/entertainment-app/src/components/SearchBar.jsx

```diff
@@ -33,15 +32,14 @@ const SearchBar = ({ selectedFilter }) =>
                 <input
                     type="text"
                     placeholder={placeholderText}
-                    value={currentSearch}
                     onChange={handleChange}
                     className={`outline-none border-b ${currentSearch ? 'border-5A698F' : 'border-transparent'} w-full px-4 py-2 text-lg bg-transparent`}
                 />
 
             </div>
-            {currentSearch && searchResults.length === 0 && (
+            {noResults && (
                 <p>
-                    No results for &quot;{currentSearch}&quot; in {placeholderText}
+                    No results for &quot;{handleSearch}&quot; in {placeholderText}
```

Comment:
Missed this from the last PR, but here I would change the no results text to something simpler like - No Results Found.